week3.js

```diff
@@ -0,0 +1,131 @@
+//String
+let myString = "Hello,this,is,a,difficult-,to-,read,sentence";
+console.log(myString);
+console.log(myString.length);
+myString = myString.replace(/,/g, " ");
+console.log(myString);
+
+//Arrays
+let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+console.log(favoriteAnimals);
+
+let favoriteAnimalsOfMauro = ["blowfish", "capricorn", "giraffe", "turtle"];
+console.log(favoriteAnimalsOfMauro);
+
+let favoriteAnimalsOfJim = ["blowfish", "meerkat", "capricorn", "giraffe", "turtle"];
+console.log("The favorite animal of Jim is added to the array.");
+console.log(favoriteAnimalsOfJim);
+console.log("The array has a length of:", favoriteAnimalsOfJim.length);
+
+let favoriteAnimalsOfJason = ["blowfish", "meerkat", "capricorn", "turtle"];
+console.log(favoriteAnimalsOfJason);
+console.log(favoriteAnimalsOfJason.length);
+console.log(favoriteAnimalsOfJason[1], "The item you are looking for is at index:2");
+
+//JavaScript
+function sum(a, b, c) {
+    return a + b + c;
+}
+console.log(sum(5, 3, 2));
+//2.
+function colorCar(color){
+    return "A" + color + "car";
+ }
+ console.log(colorCar("red"));
+//3.
+let country = {Land: "The Netherlands", Population:"17 million", Capital:"Amsterdam"};
+console.log(country);
+console.log(country.Land);
+console.log(country.Population);
+//4.

```

Comment:
The homework instruction is `Create an object and a function that takes the object as a parameter and prints out all of its properties and values.`  you created the object but not the function. Here you console.log everything yourself. Build a function and let it do the hard work. Check out  `for..in` loop as a hint and try to build a function that homework requires.  Read more about for..in loop here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

Reply:
Changed to: let country = {Land: "The Netherlands", Population:"17 million", Capital:"Amsterdam"};
for (const elements in country) {
    console.log(country.elements = country[elements]);
}

Reply:
You are almost there, the output should be  Land = The Netherlands
Population = 17 million
Capital = Amsterdam